frontend/packages/dev-console/src/components/pipelineruns/detail-page-tabs/PipelineRunLogs.tsx

```diff
@@ -138,7 +138,7 @@ class PipelineRunLogs extends React.Component<PipelineRunLogsProps, PipelineRunL
           )}
         </div>
         <div className="odc-pipeline-run-logs__container">
-          {activeItem ? (
+          {activeItem && resources[0].name ? (
```

Comment:
Checking for the name here seems a bit odd, a better approach would be to create the resources only when a pod exists.

```
const podName = taskRunFromYaml[activeItem]?.status?.podName;
const resources = taskCount > 0 && podName && [
      {
        name: podName,
        kind: 'Pod',
        namespace: obj.metadata.namespace,
        prop: `obj`,
        isList: false,
      },
    ];
```
```suggestion
          {activeItem && resources ? (
```

Reply:
Thanks! Checking during resource creation is definitely better. I've updated with your suggestion.